### PR TITLE
Unittests & bind scope in rm and rmdir functions

### PR DESCRIPTION
**Pull request by tylfin**
Error in rm and rmdir function
Binding scope to _escapeshell in rm func
Add unittest to recreate issue and cover basic initialization
Add semicolons & format issues
Update underscore -> lodash

### CODE REVIEW

**Comment by tylfin** [comment]
@Atinux reproduced the issue after you pushed your fix, added basic mocha unittests to show you.

File: index.js [hunk 1]
```diff
@@ -26,9 +26,9 @@ var spawn = require('child_process').spawn,
 */
 
 var FTP = function (options) {
-	this.initialize(options)
-	this.cmds = []
-}
+	this.initialize(options);
```
Comment (tylfin) [inline index.js]: JSHint warning fixes

File: package.json [hunk 2]
```diff
@@ -21,7 +21,7 @@
 		"node": ">=0.4.0"
 	},
 	"dependencies": {
-		"underscore": ">=1.4.4"
+    "lodash": "^4.4.0"
```
Comment (tylfin) [inline package.json]: Changed underscore -> lodash

File: test/test.js [hunk 3]
```diff
@@ -0,0 +1,128 @@
+// node-packages, requires mocha
```
Comment (tylfin) [inline test/test.js]: Created test functions to ensure underscore -> lodash preserves functionality & recreated rm/rmdir error with previous change

**Comment by Atinux** [comment]
Thanks @tylfin 
I will bump the version

**Pull request merged**